preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,41 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+import time
+
+
+def nematic_order(gsd_file):
+    # Starting timer to see how long this process takes
+    start = time.process_time()
+    s2_orders = []
+    count = 0
+    traj = gsd.hoomd.open(gsd_file)
+    nematic = freud.order.Nematic()
```

Comment:
Pass the director parameter into `freud.order.Nematic()`

Something like:

`nematic = freud.order.Nematic(np.asarray(director))`